components/server/src/workspace/gitpod-server-impl.ts

```diff
@@ -2037,7 +2046,7 @@ export class GitpodServerImpl implements GitpodServerWithTracing, Disposable {
     protected async guardTeamOperation(
         teamId: string,
         op: ResourceAccessOp,
-        fineGrainedOps: OrganizationOperation[],
+        fineGrainedOp: OrganizationOperation,
```

Comment:
Originally I thought we'd have a list of these, but I've since changed my mind. We should only reference a single operation, which in turn may require multiple different perms (like Inviting team members will require `organizaiton_metadata_read` and `organization_members_write`)